contracts/src/systems/transport/tests/travel_systems_tests.cairo

```diff
@@ -457,6 +478,11 @@ fn test_travel_hex() {
     let new_travelling_entity_position = get!(world, travelling_entity_id, Position);
     assert(new_travelling_entity_position.x == destination_coord.x, 'coord x is not correct');
     assert(new_travelling_entity_position.y == destination_coord.y, 'coord y is not correct');
+
+    let travelling_entity_wheat = get!(world, (travelling_entity_id, ResourceTypes::WHEAT), Resource);
+    assert_eq!(travelling_entity_wheat.balance, 0);
+    let travelling_entity_fish = get!(world, (travelling_entity_id, ResourceTypes::FISH), Resource);
+    assert_eq!(travelling_entity_fish.balance, 0);
```

Comment:
These should assert that 'owner_entity_id' balance is 0.
We're checking the balance of the army here, not the realm's

Reply:
ah

Reply:
fixed